Upgrade kitchen-terraform version in tutorials to 6.1

sshcherbakov opened the pull request:
The version 5.7 used in Getting Started tutorials to date doesn't work with the recent Terraform versions. Changing 5.7 to 6.1 fixes the issues.

sshcherbakov commented:
Fixes #476

aaron-lane commented on docs/tutorials/extensive_kitchen_terraform.html:
```diff
@@ -919,7 +919,7 @@ <h1 class="display-3">Extensive Kitchen-Terraform</h1>
 
 <span class="n">source</span> <span class="s2">"https://rubygems.org/"</span> <span class="k">do</span>
-  <span class="n">gem</span> <span class="s2">"kitchen-terraform"</span><span class="p">,</span> <span class="s2">"~&gt; 5.7"</span>
+  <span class="n">gem</span> <span class="s2">"kitchen-terraform"</span><span class="p">,</span> <span class="s2">"~&gt; 6.1"</span>
```
There doesn't appear to be a source change associated with this docs change 🕵🏼  Can you please make any documentation changes only in the `source` directory and then build `docs` by following the [Building Site](https://github.com/newcontext-oss/kitchen-terraform/blob/master/CONTRIBUTING.md#building-site) instructions?

aaron-lane reviewed (changes requested):
Thanks for the patch, @sshcherbakov! I only have one request below 👇🏼

sshcherbakov commented:
I removed the highlighted change and run the referenced `bundle exec middleman build --build-dir docs` command. It modified icons in the docs/ folder. Updated the PR with those changes.

aaron-lane reviewed (approved):
Thank you for the updates!
